content/admin/company/organizations.md

```diff
@@ -1,22 +1,24 @@
 ---
-description: Manage organizations for a company in Docker Admin.
+description: Manage organizations for a company in the Docker Admin Console.
 keywords: company, multiple organizations, manage organizations
 title: Manage organizations
 ---
 
 {{< include "admin-early-access.md" >}}
 
+You can manage organizations in the Docker Admin Console.
+
```

Comment:
I think we never added the extra steps around companies in Docker Hub because Admin Console has better features for company-level management. Updating to: 

`You can manage the organizations in a company in Docker Hub and the Docker Admin Console. In Docker Hub, you can manage seats and members at the organization-level. When you use the Admin Console, you can do more to manage organizations at the company-level.`